staging/src/k8s.io/legacy-cloud-providers/gce/gce_loadbalancer_internal.go

```diff
@@ -187,6 +189,10 @@ func (g *Cloud) ensureInternalLoadBalancer(clusterName, clusterID string, svc *v
 	if options.AllowGlobalAccess {
 		newFwdRule.AllowGlobalAccess = options.AllowGlobalAccess
 	}
+	if len(ports) > maxL4ILBPorts {
+		newFwdRule.Ports = nil
```

Comment:
Is it necessary to surface a warning/event that all ports are configured now despite the given ports?

Reply:
The firewall rules are added only for the specified ports, so traffic from clients to any non-listed ports will not work. We can add an Info event in any case.

Reply:
Thanks, in that sense it might be fine.